README.md

```diff
@@ -0,0 +1,97 @@
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
+## Set-up GitHub Actions
+
+Please do the following **steps in this order**:
+
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+   - **Remember** to use the file linked above
+   - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
```

Comment:
- [ ] Great job creating a `Readme` file. Kindly make sure it's following the template created by [Microverse](https://github.com/microverseinc/readme-template) and also talks about the project you're working on and not about linters 🙏.